File: keiner_lindarte_aro/instructor_henry/mis_programas/dart/3_clase_24_de_mayo/actividad.dart
import "dart:io";

void main () {

  print('Nombre:');
  String? n = stdin.readLineSync();

  print('Apellido:');
  String? ap = stdin.readLineSync();

  print('Numero:');
  int? n1 = int.parse(stdin.readLineSync()!);

  print('Numero:');
  int? n2 = int.parse(stdin.readLineSync()!);

  print('Numero:');
  int? n3 = int.parse(stdin.readLineSync()!);

  int sum = n1 + n2 + n3;

  double prom = sum/3;

  print('$n $ap');

  print('$prom');

}

File: keiner_lindarte_aro/instructor_henry/mis_programas/ruby/3_clase_24_de_mayo/actividad.rb
puts "Nombre:"
n = gets.chomp

puts "Apellido:"
ap = gets.chomp

puts "Numero:"
n1 = gets.chomp.to_i

puts "Numero:"
n2 = gets.chomp.to_i

puts "Numero:"
n3 = gets.chomp.to_i

sum = n1 + n2 + n3
prom = sum.to_f/3

puts "#{n} #{ap}"

puts prom
